### PR TITLE
tools: scripts: Makefile: Add windows support

### PR DESCRIPTION
**Pull request by scuciurean**
This PR adds the possibility to build the no-Os software in Windows environment and should be merged after #354 .

### CODE REVIEW

**Comment by scuciurean** [comment]
Update:

Rebased to master after #354 was merged.

**Comment by scuciurean** [comment]
Changelog -> v2:

1. Moved scripts to the corresponding folder

File: projects/ad9172/Makefile [hunk 1]
```diff
@@ -1,2 +1,5 @@
 TARGET := ad9172
-include ../../tools/scripts/Makefile
\ No newline at end of file
+ifeq ($(OS), Windows_NT)
```
Comment (mirceacaprioru) [inline projects/ad9172/Makefile]: commit message does not respect 75 characters/line

**Comment by scuciurean** [comment]
Changelog -> v3:

1. Fix commit message length

**Pull request merged**